### PR TITLE
Test to send emoji, file and location

### PR DESCRIPTION
**Pull request by archit2602**
### What do you want to achieve?
-Added test case for sending emoji, files, and location
-Fixed Locator values

### PR Checklist

- [ ] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- navigate to cd `playwright_test/`
- run `npx playwright test`
*Note* 
- To test code inside the tests folder make sure you have updated the file dir line 79 `chatOptions.spec.js`

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: playwright_test/playwright.config.js [hunk 1]
```diff
@@ -1,109 +1,42 @@
...
-
     {
-      name: 'firefox',
+      name: 'Desktop Safari',
       use: {
-        ...devices['Desktop Firefox'],
-      },
+        browserName: 'webkit',
+        viewport: { width: 1280, height: 720 },
+      }
```
Comment (qasinghabhi) [inline playwright_test/playwright.config.js:29]: you haven't give any permission for geo locations

File: playwright_test/tests/chatOptions.spec.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ * Created by Archit on 23/12/2011. 
+ */
+
+import { test, expect } from '@playwright/test';
+import { widgetLocators, locators } from '../locaterPackage/kmLocators';
+import { url , app_id } from '../utils/kmSecret';
+import { script } from '../utils/kmScript';
+
+let browser;
+let page;
```
Comment (qasinghabhi) [inline playwright_test/tests/chatOptions.spec.js:13]: no need to declare variable for the page and browser 
you can directly pass it in function
image

**Review by qasinghabhi — CHANGES_REQUESTED** [review]

File: playwright_test/playwright.config.js (hunk 1 above)
Comment (archit2602) [reply playwright_test/playwright.config.js:29]: not required for WebKit

File: playwright_test/tests/chatOptions.spec.js (hunk 2 above)
Comment (archit2602) [reply playwright_test/tests/chatOptions.spec.js:13]: ya, I forget to delete the browser part that I will delete in the upcoming PR also if I delete the page test will start falling.

**Review by devashishmamgain — APPROVED** [review]

**Review by vipinkom1 — APPROVED** [review]

**Pull request merged**